compiler/hash-lexer/src/lib.rs

```diff
@@ -964,7 +963,7 @@ impl<'a> Lexer<'a> {
             return self.emit_fatal_error(
                 None,
                 LexerErrorKind::UnclosedStringLit,
-                Span::new(start, self.offset.get()),
+                ByteRange::new(start, self.offset.get()),
```

Comment:
Is this truly a byte range or a codepoint range?

Reply:
It is kind of both.

Reply:
Maybe it is better to call it `CharacterRange` or something like that?

Reply:
hmmm but it does specifically refer to bytes, the `RowColRange` is more of a character range.